tpr/backoff.go

```diff
@@ -0,0 +1,25 @@
+package tpr
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
```

Comment:
Agreed, but why don't we then just use https://godoc.org/github.com/cenkalti/backoff#ExponentialBackOff and set `MaxElapsedTime` to 90 seconds? This is what I mean. We can just configure what the lib provides and don't need to implement the interface on our own.